views/LocationListView/LocationListView.tsx

```diff
@@ -1,21 +1,22 @@
-import { useEffect, useState } from 'react';
+import { FC, useCallback } from 'react';
 import React from 'react-native';
+import { useFocusEffect } from '@react-navigation/native';
 import { Button, ListItem, Typography } from '../../.storybook/stories';
 import { LayoutProvider, NavbarWithLogout } from '../../components';
 import { LoadingWrapper } from '../../components/LoadingWrapper';
-import { useAppNavigation } from '../../hooks';
-import { resolveUserID } from '../../utils';
-import { BOT_NAMES } from '../../utils/env';
+import {
+  useAppNavigation,
+  useBotsNames,
```

Comment:
Hmm isn't this hook will be used only in this view? Maybe you can place it in the hooks folder inside a LocationListView folder